gnocchi/rest/__init__.py

```diff
@@ -665,6 +667,8 @@ def post(self):
         except indexer.NoSuchResource as e:
             abort(404, e)
 
+        return r.metrics
```

Comment:
There's no order set by default. So, *by chance* it might always been the same order, but it also can depends on whetever it runs on MySQL or PostgreSQL. You can't assume in the test that the order is going to be always the same. Especially because maybe the next release of $RDBMS will break that assumption.

Now, what you can do is sort in the `response_json_paths`. This is documented here:

http://gabbi.readthedocs.io/en/latest/jsonpath.html

Basically you can do this:

```
response_json_paths:
  $[/name][1].name: foobar
```

If your metric list is "foobar, cpu_util" for example (you'll need to check what the full list is, then "foobar" will be alphabetically after "cpu_util" when sorting by name (the `[/name]` in the JSON path expression.